xml/ha_rear.xml

```diff
@@ -445,10 +445,11 @@
   </para>
 
   <para>
-   A default &rear; configuration for reference is available at
-   <filename>/usr/share/rear/conf/default.conf</filename>. Other example
-   configurations (<filename>*example.conf</filename>) are available in the
-   <filename>example</filename> subdirectory. 
+   All &rear; configuration variables and their default values are set in
+   <filename>/usr/share/rear/conf/default.conf</filename>.
+   Some example files (<filename>*example.conf</filename>) for user configurations
+   (for example, what is set in <filename>/etc/rear/local.conf</filename>)
+   are available in the <filename>examples</filename> subdirectory.
    Find more information in the &rear; man page.
   </para>
 
```

Comment:
Thanks for the hints! I rephrased it a bit to match our style:

```
You should start with a matching example config file as template
and adapt it as needed to create your particular configuration file. 
Copy various options from several example config files and paste them
into your specific <filename>/etc/rear/local.conf</filename> file that
matches your particular system.
Do not use original example configuration files, because they simply
provide an overview of variables that can be used for specific setups.
```

I hope I didn't introduce a different sense. Tell me what you think about it. 😉